spec/worker_spec.rb

```diff
@@ -47,23 +44,30 @@
       EOJ
     end
 
-    let(:source_types_1_sources_response) do
+    let(:topology_sources_response) do
       <<~EOJ
         {
           "links": {},
           "data": [
-            {"id":"1","source_type_id":"1","name":"mock-source","uid":"cacebc33-1ed8-49d4-b4f9-713f2552ee65","tenant_id":"1"},
-            {"id":"2","source_type_id":"1","name":"OCP","uid":"31b5338b-685d-4056-ba39-d00b4d7f19cc","tenant_id":"1"}
+            {"id":"1","uid":"cacebc33-1ed8-49d4-b4f9-713f2552ee65","tenant_id":"1"},
+            {"id":"2","uid":"31b5338b-685d-4056-ba39-d00b4d7f19cc","tenant_id":"1"}
           ]
         }
       EOJ
     end
 
-    let(:source_types_2_sources_response) do
+    let(:sources_1_response) do
       <<~EOJ
         {
-          "links": {},
-          "data": []
+          "id":"1","source_type_id":"1","name":"mock-source","uid":"cacebc33-1ed8-49d4-b4f9-713f2552ee65","tenant":"#{user_tenant_account}"
```

Comment:
Does the sources API not have paging?

Reply:
It does, but this is a get on an id not the collection